defaults/main.yml

```diff
@@ -233,6 +233,16 @@ pki_acme_challenge_dir: '{{ (ansible_local.nginx.acme_root
                              else "/srv/www/sites/acme/public") +
                              "/.well-known/acme-challenge" }}'
 
+
+
+# .. envvar:: pki_create_acme_challenge
+#
+# Whether the directory where the ACME client should store responses
+# should be created.
+pki_create_acme_challenge_dir: '{{
+    (ansible_local|d() and ansible_local.nginx|d()
+     and  ansible_local.nginx.acme|d() and
+          ansible_local.nginx.acme)|bool }}'
```

Comment:
I'm not sure what happens if the `ansible_local.nginx.*` dictionary is not set. Perhaps go with something like this instead:
```yaml
pki_create_acme_challenge_dir: '{{ True if (ansible_local|d() and ansible_local.nginx|d() and
                                            ansible_local.nginx.acme|d() and
                                            ansible_local.nginx.acme|bool) else False }}'
```